### DISCUSSION TITLE
It's brilliant - but the Chrome tab runs out of memory after a while

### DISCUSSION THREAD

**Discussion by gertalot** [najdek/f1tv_plus#18] [Q&A]
This is an amazing tool, thanks so much for putting it together! I run a custom 5-window setup (4x 20% W x 25% H on the left, and a 80% W x 100% H main window on the right), but after playing for a while I get the "Aw snap!" from Chrome with an out of memory error.

Have you seen these out of memory issues?

Edit: Having the "memory" tab of the inspector open, it looks like whenever you switch a frame to a different camera, it adds 2 new VM instances but doesn't remove any old ones, hmm. Could that be the issue?

FWIW, here's my 5 frame config:

```
        [
            [20, 0, 80, 100],
            [0, 0, 20, 25],
            [0, 25, 20, 25],
            [0, 50, 20, 25],
            [0, 75, 20, 25]
        ],
```

**Comment by zierbeek** [comment]
same !
